1080i/Info.xml

```diff
@@ -258,16 +258,41 @@
                         <param name="textcolor" value="$PARAM[colordiffuse]_90" />
                     </include>
 
+                    <!-- Duration Info -->
                     <include content="Info_Line_Label" condition="!$PARAM[player]">
                         <param name="controltype" value="$PARAM[controltype]" />
                         <param name="label" value="$INFO[$PARAM[container]ListItem.Duration(h),, $LOCALIZE[31074] ]$INFO[$PARAM[container]ListItem.Duration(m),, $LOCALIZE[31073]]" />
-                        <param name="visible" value="!String.IsEmpty($PARAM[container]ListItem.Duration) + !String.IsEqual(ListItem.DBType,artist) + !String.IsEqual(ListItem.DBType,album) + !String.IsEqual(ListItem.DBType,song) + !Window.IsVisible(MyPVRGuide.xml)" />
+                        <param name="visible" value="!String.IsEmpty($PARAM[container]ListItem.Duration) + !String.IsEqual(ListItem.DBType,artist) + !String.IsEqual(ListItem.DBType,album) + !String.IsEqual(ListItem.DBType,song) + !Window.IsVisible(MyPVRGuide.xml) + !String.IsEqual($PARAM[container]ListItem.Duration(h), 0)" />
```

Comment:
Wouldn't you also need ` + !String.IsEqual($PARAM[container]ListItem.Duration(m), 0)` here? Otherwise it is going to display at the same time as the hours only one.

Reply:
Didn't think about that :) Will do now